src/services/ledger/schema.types.d.ts

```diff
@@ -0,0 +1,6 @@
+interface TransactionMetadataRecord {
+  _id: ObjectId
+
+  hash?: string
```

Comment:
`hash` and `revealedPreImage` are specified as optional here but not in the schema:
```
  {
     hash: String,
     revealedPreImage: {
       type: String,
       index: true,
     },
   },
   ```
   Or is optional the default?

Reply:
Would have to confirm, but I believe optional is the default and we add `required: true` to make it non-optional